src/main/resources/db/migration/development/U99__seed_data.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO court (id, name, court_code) VALUES (1142407, 'Sheffield Magistrates Court', 'SHF');
```

Comment:
An undo migration would normally undo the effects of the original migration so I'd expect this to be along the lines of `DELETE FROM court_case where * id in [blah....]`. Not sure I understand why we need it so I'll give you a shout when we're both in if that's cool.